audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.java

```diff
@@ -1,51 +1,46 @@
 package com.sequenceiq.cloudbreak.audit.converter;
 
-import static com.sequenceiq.cloudbreak.util.ConditionBasedEvaluatorUtil.doIfTrue;
-
-import org.apache.commons.lang3.StringUtils;
-import org.springframework.stereotype.Component;
-
 import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.converter.builder.AttemptAuditEventResultBuilderProvider;
 import com.sequenceiq.cloudbreak.audit.model.AttemptAuditEventResult;
-import com.sequenceiq.cloudbreak.audit.model.ResultApiRequestData;
 import com.sequenceiq.cloudbreak.audit.model.ResultEventData;
-import com.sequenceiq.cloudbreak.audit.model.ResultServiceEventData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
 
 @Component
 public class AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter {
 
-    public AuditProto.AttemptAuditEventResult convert(AttemptAuditEventResult source) {
-        AuditProto.AttemptAuditEventResult.Builder attemptAuditEventResultBuilder = prepareBuilderForCreateAuditEvent(source);
-        updatResultEventData(attemptAuditEventResultBuilder, source.getResultEventData());
-        return attemptAuditEventResultBuilder.build();
+    private static final Logger LOGGER = LoggerFactory.getLogger(AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.class);
+
+    private final Map<Class, AttemptAuditEventResultBuilderUpdater> builderUpdater;
+
+    private final AttemptAuditEventResultBuilderProvider builderProvider;
+
+    public AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter(Map<Class, AttemptAuditEventResultBuilderUpdater> builderUpdater,
+                    AttemptAuditEventResultBuilderProvider builderProvider) {
+        this.builderUpdater = builderUpdater;
+        this.builderProvider = builderProvider;
```

Comment:
what is the reason behind this? besides, it is not consistent with the other converter class

Reply:
the reason behind this is to make the testing easier by abstracting these builders - like the AttemptAuditEventResult's builder - since they're coming outside from our codebase and the earlier tests were relying on their functionality. but it's easy to acknowledge that if somebody would have changed the functionality of some these builders behind the scene, the tests would fail even if our logic would be the same